EIPS/eip-2121.md

```diff
@@ -0,0 +1,81 @@
+---
+eip: 2121
```

Comment:
I believe this number is taken by #2121. This EIP should be 2330.

Reply:
Ah, got it -- will update